### PR TITLE
feat: add flip drop-down (video flags)

### PR DESCRIPTION
**Pull request by hagemt**
Handles #4

### CODE REVIEW

**Comment by hagemt** [comment]
image

File: .eslintrc.json [hunk 1]
```diff
@@ -7,5 +7,7 @@
     "ecmaVersion": 2020,
     "sourceType": "module"
   },
+  "reportUnusedDisableDirectives": true,
+  "root": true,
```
Comment (EvanHahn) [inline .eslintrc.json]: Could you revert this? I don't think they're really related to this PR.

I'd love to have them in a follow-up change, though!

File: package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "private": true,
   "type": "module",
   "scripts": {
+    "dev": "serve src",
```
Comment (EvanHahn) [inline package.json]: Similar to the above: could you revert the changes to this file?

**Review by EvanHahn — CHANGES_REQUESTED** [review]
Looks great other than some extraneous changes. Thanks for doing this!

**Comment by hagemt** [comment]
I kept just the changes in `src` as requested, even though I'd politely suggest you also take the JSON changes while we're here, since it's the bare minimum to dev without a few minor annoyances. Knowing me, I'm probably not going to check back on this in all honesty. 🙂 Recently, ESLint wants root: true, to avoid the provenance of config in home directories.

P.S. Thanks for `helmet` and your other modules NPM. I didn't realize you're also the author of that, and on human note: I appreciate you as a maintainer. Cheers 🎉

**Review by EvanHahn — APPROVED** [review]
Thanks for doing this!

**Comment by EvanHahn** [comment]
I re-added your `.eslintrc` changes in 3f0b5be254fb13b6c0963002d271fd4f660e63aa and 6d16fae67dfda7d8c762956e4142e56d40253f73.

Thanks again! I'll deploy this soon.

**Pull request merged**